DataFormats/MuonReco/interface/Muon.h

```diff
@@ -179,15 +179,20 @@ namespace reco {
 
 
     /// define arbitration schemes
-    enum ArbitrationType { NoArbitration, SegmentArbitration, SegmentAndTrackArbitration, SegmentAndTrackArbitrationCleaned, RPCHitAndTrackArbitration };
+    enum ArbitrationType { NoArbitration, SegmentArbitration, SegmentAndTrackArbitration, SegmentAndTrackArbitrationCleaned,
+			   RPCHitAndTrackArbitration, GEMSegmentAndTrackArbitration, ME0SegmentAndTrackArbitration };
     
     ///
     /// ====================== USEFUL METHODs ===========================
     ///
-    /// number of chambers (MuonChamberMatches include RPC rolls)
+    /// number of chambers (MuonChamberMatches include RPC rolls, GEM and ME0 segments)
     int numberOfChambers() const { return muMatches_.size(); }
     /// number of chambers not including RPC matches (MuonChamberMatches include RPC rolls)
     int numberOfChambersNoRPC() const;
+    /// number of chambers not including GEM matches (MuonChamberMatches include RPC rolls, and also GEM)
+    int numberOfChambersNoGEM() const;    
+    /// number of chambers not including ME0 matches (MuonChamberMatches include RPC rolls, and also ME0)
+    int numberOfChambersNoME0() const;    
```

Comment:
the purpose of numberOfChambersNoRPC was to capture the number of CSC-or-DT chambers.
I picked the name in 2011 not thinking of the future upgrades for a somewhat obvious reason.

numberOfChambersNoGEM and numberOfChambersNoME0 don't fit this purpose.
To make a more meaningful update, rename numberOfChambersNoRPC to be numberOfChambersCSCorDT.